### PR TITLE
 ESLintBear: Fix KeyErrors for file level errors

### PR DESCRIPTION
**Pull request by jayvdb**
Fixes https://github.com/coala/coala-bears/issues/2533

### CODE REVIEW

File: tests/js/ESLintBearTest.py [hunk 1]
```diff
@@ -67,3 +77,36 @@
...
+    def test_lint_config_file(self):
+
+        self.maxDiff = None
+        config_filename = os.path.join(test_dir, '.eslintrc.js')
+
+        self.section.append(Setting('eslint_config', config_filename))
+
+        expected = Result.from_values(
+            'ESLintBear',
+            'File ignored by default.  Use a negated ignore pattern '
```
Comment (jayvdb) [inline tests/js/ESLintBearTest.py]: Note ive created https://github.com/coala/coala-bears/issues/2607 to determine how we handle this properly.

**Comment by blazeu** [comment]
Why is there a 2 space between sentences in commit https://github.com/coala/coala-bears/pull/2608/commits/71c67c0aa6cc319d49901a0a690f0b31bef1ae89 ?

**Comment by jayvdb** [comment]
Because that is proper English :P Fixed.

**Comment by blazeu** [comment]
ack https://github.com/coala/coala-bears/commit/e934f069e0a1f79aa162ddcf9f9fb3bec950fe2c https://github.com/coala/coala-bears/commit/9e3c7ba266a579d846471f2b07db11fc96ddb9e0

**Comment by yukiisbored** [comment]
@gitmate-bot ff

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**